Examples/KlaviyoSwiftExamples/Shared/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>Klaviyo_badge_autoclearing</key>
+	<false/>
+	<key>Klaviyo_App_Group</key>
+	<string></string>
```

Comment:
is this value supposed to be empty?

Reply:
kinda? it needs to be set up by the dev, not sure if we should put a placeholder value?

Reply:
since it's an example app, I think it should be fully functional out of the box. Will the example app function as expected if this value is blank? If not, I think we should provide a value here. 

It looks like the naming convention for app groups is `group.<bundle identifier>`. So I think this value can be `<string>group.$(PRODUCT_BUNDLE_IDENTIFIER)</string>`